builtin/v14/miner/miner_types.go

```diff
@@ -496,3 +496,32 @@ type SectorReturn = []PieceReturn
 
 // PieceReturn represents a result for each piece for the sector that was notified.
 type PieceReturn = bool // Accepted = true
+
+// SectorNIActivationInfo is the information needed to activate a sector with a "zero" replica.
+type SectorNIActivationInfo struct {
+	SealingNumber abi.SectorNumber // Sector number used to generate replica id
+	SealerID      abi.ActorID      // Must be set to ID of receiving actor for now
+	SealedCID     cid.Cid          // CommR
+	SectorNumber  abi.SectorNumber // unique id of sector in actor state
+	SealRandEpoch abi.ChainEpoch
+	Expiration    abi.ChainEpoch
+}
+
+// ProveCommitSectorsNIParams is the parameters for non-interactive prove committing of sectors
+// via the miner actor method ProveCommitSectorsNI.
+type ProveCommitSectorsNIParams struct {
```

Comment:
We probably want to add `ProvingDeadline uint64` eventually in here as well. Given that we want to land https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0092.md together with NI-PoRep.

But maybe worth waiting until that has moved to Last Call